### DISCUSSION TITLE
l-form-item使用报错

### DISCUSSION THREAD

**Discussion by MinCpopo** [TaleLin/lin-ui#1222] [求助] [answered]
在使用l-form-item时，页面报错“表单项不存在name”，同一页面上定义了多个l-form-item，但只有这一个报错，不知道是不是和该l-form-item下是我自定义的内容有关，下面是我代码的截图
![image](https://user-images.githubusercontent.com/42403875/103184965-419e1a80-48f5-11eb-8cb8-d61a91648647.png)

![image](https://user-images.githubusercontent.com/42403875/103184974-4fec3680-48f5-11eb-850d-9ce4425ae839.png)

**Comment by juzi214032** [comment, marked as answer]
这段代码是在页面 wxml 中还是在自定义组件中的？
